docs/extensions/create-specification-builder.md

```diff
@@ -5,4 +5,70 @@ parent: Extensions
 nav_order: 2
 ---
 
-How to create your own specification builder
+
+# How to create your own specification builder
+How to create your own specification builder 
```

Comment:
```suggestion

How to create your own specification builder 
```

Reply:
Also, although the title is all about "How" it would be great to say "Why" one would want/need to do this somewhere at the top, too.